frontend/src/components/podcast/podcast.tsx

```diff
@@ -94,12 +93,21 @@ const Podcast: React.FC = () => {
               label={podcast.name}
               className={styles.imageWrapper}
             />
-            {podcast.userId === userId && (
-              <Button
-                className={styles.addEpisodeLink}
-                label="Add episode"
-                href={`${AppRoute.PODCASTS}/${podcast.id}${AppRoute.EPISODES_EDIT}`}
-              />)
+            {podcast.userId === userId
```

Comment:
user can have both these buttons

Reply:
> user can have both these buttons

here I check the owner of the podcast and if not then show the follow button